apps/src/applab/applab.js

```diff
@@ -618,7 +621,7 @@ Applab.init = function (config) {
   config.level.levelBlocks = config.dropletConfig.blocks;
 
   getStore().dispatch(actions.changeInterfaceMode(
-    Applab.startInDesignMode() ? ApplabInterfaceMode.DESIGN : ApplabInterfaceMode.CODE));
+    (!teacherViewStudentWork && Applab.startInDesignMode()) ? ApplabInterfaceMode.DESIGN : ApplabInterfaceMode.CODE));
```

Comment:
While you were in here, did you figure out why we have to `changeInterfaceMode` twice during initialization?  Seems like we should be able to remove one of these.